spec/models/step_by_step_page_spec.rb

```diff
@@ -309,7 +331,7 @@
     let(:step_by_step_page) { create(:published_step_by_step_page) }
 
     before do
-      allow(step_by_step_page).to receive(:links_checked?) { true }
+      allow(step_by_step_page).to receive(:links_checked_since_last_update?) { true }
     end
 
     it 'can be published if it has a draft, is not scheduled, all steps have content and links have been checked' do
```

Comment:
I'd suggest updating this (and all other similar test cases) to say something like `...links have been checked since last update` (in commit c48bea5f91a0c1de86be463a79cfa44d09bfea3f) since you're now testing `links_checked_since_last_update?` rather than `links_checked?`.